lib/matplotlib/_api/__init__.py

```diff
@@ -139,6 +140,35 @@ def check_in_list(values, /, *, _print_supported_values=True, **kwargs):
             raise ValueError(msg)
 
 
+def check_in_keys(value, mappable, *,
```

Comment:
This is basically a dupe of `check_getitem`, except with an additional error message?

Reply:
Ha, thanks, for some reason I didn't spot `check_getitem`. I shall modify that instead.